### ISSUE TITLE
Bug Report for winME

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#4110]
Migrated from [rt.perl.org#7147](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=7147) (status was 'resolved')

Searchable as RT7147$

**Comment by p5pRT** [comment]
### From rcr@rcrrich.com

#### Created by rcrrich@rcrrich.com


C&#8203;:\\httpd\\Cgi\-Bin\\reseller\-demo>perl sell\.pl
Pragma&#8203;: no\-cache
Content\-type&#8203;:text/html
 
install\_driver\(c\) failed&#8203;: Can't locate DBD/c\.pm in @&#8203;INC \(@&#8203;INC contains&#8203;:
C&#8203;:/Perl/
lib C&#8203;:/Perl/site/lib \.\) at \(eval 1\) line 3\.
Perhaps the DBD&#8203;::c perl module hasn't been fully installed\,
or perhaps the capitalisation of 'c' isn't right\.
Available drivers&#8203;: ADO\, ExampleP\, Multiplex\, ODBC\, Proxy\, mysql\.
 at module\.pl line 142
 
C&#8203;:\\httpd\\Cgi\-Bin\\reseller\-demo>
 
 
 

<details><summary>Perl Info</summary>

```

Flags:
    category=library
    severity=critical

Site configuration information for perl v5.6.1:
 
Summary of my perl5 (revision 5 version 6 subversion 1) configuration:
  Platform:
    osname=MSWin32, osvers=4.0, archname=MSWin32-x86-multi-thread
    uname=''
    config_args='undef'
    hint=recommended, useposix=true, d_sigaction=undef
    usethreads=undef use5005threads=undef useithreads=define
usemultiplicity=define
    useperlio=undef d_sfio=undef uselargefiles=undef usesocks=undef
    use64bitint=undef use64bitall=undef uselongdouble=undef
  Compiler:
    cc='cl', ccflags ='-nologo -O1 -MD -DNDEBUG -DWIN32 -D_CONSOLE
-DNO_STRICT -DHAVE_DES_FCRYPT  -DPERL_IMPLICIT_CONTEXT
-DPERL_IMPLICIT_SYS -DPERL_MSVCRT_READFIX',
    optimize='-O1 -MD -DNDEBUG',
    cppflags='-DWIN32'
    ccversion='', gccversion='', gccosandvers=''
    intsize=4, longsize=4, ptrsize=4, doublesize=8, byteorder=1234
    d_longlong=undef, longlongsize=8, d_longdbl=define, longdblsize=10
    ivtype='long', ivsize=4, nvtype='double', nvsize=8, Off_t='off_t',
lseeksize=4
    alignbytes=8, usemymalloc=n, prototype=define
  Linker and Libraries:
    ld='', ldflags ='-nologo -nodefaultlib -release
-libpath:"C:\Perl\lib\CORE"  -machine:x86'
    libpth="C:\Perl\lib\CORE"
    libs=  oldnames.lib kernel32.lib user32.lib gdi32.lib winspool.lib
comdlg32.lib advapi32.lib shell32.lib ole32.lib oleaut32.lib
netapi32.lib uuid.lib wsock32.lib mpr.lib winmm.lib  version.lib
odbc32.lib odbccp32.lib msvcrt.lib
    perllibs=  oldnames.lib kernel32.lib user32.lib gdi32.lib
winspool.lib  comdlg32.lib advapi32.lib shell32.lib ole32.lib
oleaut32.lib  netapi32.lib uuid.lib wsock32.lib mpr.lib winmm.lib
version.lib odbc32.lib odbccp32.lib msvcrt.lib
    libc=msvcrt.lib, so=dll, useshrplib=yes, libperl=perl56.lib
  Dynamic Linking:
    dlsrc=dl_win32.xs, dlext=dll, d_dlsymun=undef, ccdlflags=' '
    cccdlflags=' ', lddlflags='-dll -nologo -nodefaultlib -release
-libpath:"C:\Perl\lib\CORE"  -machine:x86'
 
Locally applied patches:
    ACTIVEPERL_LOCAL_PATCHES_ENTRY
 

@INC for perl v5.6.1:
    C:/Perl/lib
    C:/Perl/site/lib
    .
 

Environment for perl v5.6.1:
    HOME (unset)
    LANG (unset)
    LANGUAGE (unset)
    LD_LIBRARY_PATH (unset)
    LOGDIR (unset)
 
PATH=C:\WINDOWS;C:\WINDOWS\COMMAND;C:\Perl\bin;C:\PROGRA~1\SYMANTEC\PCAN
YW~1\;C:\HTTPD\PHP;C:\HTTPD\CGI-BIN\RESELLER-DEMO;C:\MYSQL\BIN
    PERL_BADLANG (unset)
    SHELL (unset)
 
 
 
_________________________________________________________________
Richard C Rossy
(305) 823-0358
Smart <http://www.mysmartnetworker.com/?1077>  Networker
IT IS THE ONLY PLACE I KNOW OF WHERE WITH A SMALL START UP FEE
YOU CAN BUILD MASSIVE WEALTH! 

Click to add my contact info to your organizer:
http://my.infotriever.com/rcrwbn
 

```
</details>

**Comment by p5pRT** [comment]
### From rcr@rcrrich.com

[image001.gif](https://rt-archive.perl.org/perl5/Ticket/Attachment/17439/18120/image001.gif)

**Comment by p5pRT** [comment]
### From [Unknown Contact. See original ticket]

> C&#8203;:\\httpd\\Cgi\-Bin\\reseller\-demo>perl sell\.pl
> Pragma&#8203;: no\-cache
> Content\-type&#8203;:text/html
>  
> install\_driver\(c\) failed&#8203;: Can't locate DBD/c\.pm in @&#8203;INC \(@&#8203;INC contains&#8203;:
> C&#8203;:/Perl/
> lib C&#8203;:/Perl/site/lib \.\) at \(eval 1\) line 3\.
> Perhaps the DBD&#8203;::c perl module hasn't been fully installed\,
> or perhaps the capitalisation of 'c' isn't right\.
> Available drivers&#8203;: ADO\, ExampleP\, Multiplex\, ODBC\, Proxy\, mysql\.
>  at module\.pl line 142

This looks more like a problem with a script you have than a bug in 
Perl\.

Cheers\,
Philip

**Comment by p5pRT** [comment]
### From @JohnPeacock

> his is a bug report for perl from rcrrich@&#8203;rcrrich\.com\,
> generated with the help of perlbug 1\.33 running under perl v5\.6\.1\.
>  
>  
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> \[Please enter your report here\]
> C&#8203;:\\httpd\\Cgi\-Bin\\reseller\-demo>perl sell\.pl
> Pragma&#8203;: no\-cache
> Content\-type&#8203;:text/html
>  
> install\_driver\(c\) failed&#8203;: Can't locate DBD/c\.pm in @&#8203;INC \(@&#8203;INC contains&#8203;:
> C&#8203;:/Perl/
> lib C&#8203;:/Perl/site/lib \.\) at \(eval 1\) line 3\.
> Perhaps the DBD&#8203;::c perl module hasn't been fully installed\,
> or perhaps the capitalisation of 'c' isn't right\.
> Available drivers&#8203;: ADO\, ExampleP\, Multiplex\, ODBC\, Proxy\, mysql\.
>  at module\.pl line 142
>  
> C&#8203;:\\httpd\\Cgi\-Bin\\reseller\-demo>
>  
>  
>  
> \[Please do not change anything below this line\]
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-


Several things&#8203;:

1\.  Don't send HTML mail to any maillist; it is rude and will most
likely turn off anyone who might be inclined to help you\.

2\.  Send to the correct maillist; this is a list for developers of 
Perl\, not a general help desk\.  Your question should be directed at
the dbi\-users list\, see

&nbsp;    http&#8203;://www\.isc\.org/dbi\-lists\.html

for instructions for joining the list\.

3\.  Show the code; you didn't show us any code that would help us
diagnose your problem\.  I suspect that your DBI\->connect\(\) call is
referring to an unknown driver \(i\.e\. one you haven't installed\) or
is just malformed\.  Again\, the dbi\-users list is the place to ask\.

HTH

John Peacock

p\.s\. ATTN&#8203;: BUG MODERATORS \- close the bug
